docs/spec/data-plane.md

```diff
@@ -91,6 +91,25 @@ delivered.
 CloudEvents received by Sink MAY have
 [Distributed Tracing Extension Attribute](https://github.com/cloudevents/spec/blob/v1.0/extensions/distributed-tracing.md).
 
+### HTTP Attributes
+
+An event sender, including Source and Broker and Channel, MUST include
+all the non-default HTTP attributes with header key `K-Eventing-Http-Attr` in
+every event delivery to specify
+what HTTP features it can support. Without a certain attribute, the receiver
+MUST assume the corresponding HTTP feature is not supported. A list of HTTP
+attributes can be found below:
+
+| Attributes         | HTTP Feature Description                                                  |
+| ------------------ | ------------------------------------------------------------------------- |
+| `callable`         | If the event sender supports event reply in HTTP response.                |
+
+An example is that a broker supporting event reply MUST send events with
```

Comment:
this `MUST` is not spec language, this is an example. You don't tend to give spec language in a clarifying paragraph with an example.

Reply:
Done